decidim-meetings/app/helpers/decidim/meetings/admin/filterable_helper.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Meetings
+    module Admin
+      module FilterableHelper
+        def extra_dropdown_submenu_options_items(_filter, _i18n_scope)
+          options = nil
+          # options = case filter
```

Comment:
⚠️ Why is this commented? Please, uncomment it if it's going to be used or just return `[]` if not.